plugins/fclrave/fclspace.cpp

```diff
@@ -106,7 +102,11 @@ void FCLSpace::ReloadKinBodyLinks(KinBodyConstPtr pbody, FCLKinBodyInfoPtr pinfo
                 // or, should the collision report store names of body, link, and geom?
```

Comment:
I think now we can delete this comment about having no geometry name.

Reply:
right, i removed resolved todo comment.
meanwhile, i keep the unresolved comment about putting geometry group information somewhere.